_posts/2016-04-25-episerver-developer-meetup-helsinki.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Episerver developer meetup at Helsinki
+author: Rinorragi
+excerpt: What happened in the first Episerver developer meetup held in Helsinki
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- security
+---
+Last week Episerver organized a meetup for Finnish developers in Helsinki. Meetup consisted of pizza, beer and presentations. I loved the meetup and I think that the format was a perfect match to have a cozy atmosphere around the Episerver topic. Only thing that could be done better (for the next time) would be to have more time at the end for the networking! Which just means that the actual event has been great success! 
+
+## R.I.P XForms 
+
+Mikko Huilaja from Solita had a hour long presentation about new Episerver Forms. He went through how they work, how you can create them, edit them, and also how to extend them. I haven't had the time before to take look on the new Episerver Forms before but now I feel I would be ready to implement them any day. Thanks Mikko for this great deep dive into the new Episerver Forms!
+[Slides](http://www.slideshare.net/huilaaja/Episerver-forms-fi)
+
+## Commerce promotion engine 
+
+Quan Mai gave us a sneak peek into the upcomming promotion engine enhancement. If I understood correctly, the promotion engine might be included in next Commerce major release. The new system was IContent based and it was promised to be a lot faster than the old system. Debugging and extending it seemed to be thought in a good manner. All this means that there will be even less need to use the old Commerce Manager in the future. 
```

Comment:
"If I understood correctly, the promotion engine might be included in next Commerce major release"
Onko tässä hiukan tuplakonditionaalia. Eli koska "jossittelu" hoidetaan ekassa lauseessa, toinen lause voisi olla vain "the promotion engine **will ** be included..."

"system was" -> "system **is**"?